### DISCUSSION TITLE
Update query failing randomly, No "X" found

### DISCUSSION THREAD

**Discussion by ddm50** [prisma/prisma#22844] [Q&A]
### Question

Updating record on table Y leads to No "X" found error, table Y has a relationship to X so I'm guessing that's the reason? And yet in a different project updating the same record works, no issues so it cannot be reproduced.

Supabase shows no errors so clearly Prisma is screwing with me, it probably thinks the query failed for some reason.
Enabled debugging, absolutely nothing is shown when making the query update so... 

```

await prisma.payments.update({
            where: {
                id: 375,
            },
            data: {
                status: 'PAID',
                paid_date: new Date(),
            }
        })
``` = No "User" found

**Comment by nurul3101** [comment]
Hello @ddm50 👋 

Could you please share the entire error message that you are receiving?
It would be helpful if you could share the models with us so that I can reproduce this and check.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
